system/jlib/jmalloc.cpp

```diff
@@ -1178,6 +1178,8 @@ class CSuperAllocator: public CInterface, implements IAllocator
             *   3. Put a 'free' tag on the end.
             *   4. Change the size recorded in the block.
             */
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
```

Comment:
Chose to temporarily ignore the maybe-uninitialized error being through when building against glibc v2.24.